akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentActorCompileOnlyTest.scala

```diff
@@ -404,7 +394,7 @@ object PersistentActorCompileOnlyTest {
     private val commandHandler: CommandHandler[Command, Event, State] = CommandHandler.command {
       case Enough ⇒
         Effect.persist(Done)
-          .andThen((_: State) ⇒ println("yay"))
+          .thenRun((_: State) ⇒ println("yay"))
```

Comment:
Oh no, we still need the type here, I was hoping this maybe solved #25383

Reply:
Yeah it is because Effect.persist doesn't know what it's `State` param is so it is inferred to Nothing